test/dialog_test.rb

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env rspec
+# encoding: utf-8
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "test_helper"
+
+require "yast2/system_service"
+require "cwm/service_widget"
+
+Yast.import "DhcpServer"
+
+describe "DhcpServerDialogsInclude" do
+  class TestDialog
+    include Yast::I18n
+    include Yast::UIShortcuts
+
+    def initialize
+      @HELPS = {}
+      Yast.include self, "dhcp-server/widgets.rb"
+      Yast.include self, "dhcp-server/dialogs.rb"
+    end
+  end
+
+  let(:widget) { instance_double(::CWM::ServiceWidget, cwm_definition: {}) }
+  let(:service) { instance_double(Yast2::SystemService, save: true, currently_active?: true) }
+
+  before do
+    allow_any_instance_of(TestDialog).to receive(:fun_ref)
```

Comment:
I am not a big fan of `allow_any_instance_of`. Actually, its usage is [discouraged](https://relishapp.com/rspec/rspec-mocks/docs/working-with-legacy-code/any-instance). What about setting this expectation in the subject (if possible) or just adding the method to the `TestDialog` class? Anyway, we are working with legacy code, so it is not that bad.

Reply:
I did not know that. Thank you.

I will to update/improve it